index.html

```diff
@@ -2329,7 +2329,68 @@ <h2 id="OfflineAudioCompletionEvent">
           value explicitly. <em>block-size</em> is defined to be 128
           sample-frames which corresponds to roughly 3ms at a sample-rate of
           44.1KHz.
-        </p>
+        </p><a>AudioNode</a>s can be created in two ways:
+        <ul>
```

Comment:
Since you say "First", "Second" below for each case, I think you should either make this an ordered list and remove "First", "Second" below.